syntax.php

```diff
@@ -30,7 +30,11 @@ public function getSort() {
      * @param string $mode Parser mode
      */
     public function connectTo($mode) {
-        $this->Lexer->addSpecialPattern('\\{\\{top\\}\\}', $mode, 'plugin_top');
+        $this->Lexer->addEntryPattern('\\{\\{top\\|?(?=.*?\\}\\})', $mode, 'plugin_top');
+    }
+
+    function postConnect() {
+        $this->Lexer->addExitPattern('.*?\\}\\}', 'plugin_top');
     }
```

Comment:
It would be simpler to contiue using the SpecialPattern (just extend it to match the whole syntax) and then parse out parameters in handle(). Entry Exit should really only be needed when accepting other plugin syntax with your own plugin.